mmv1/third_party/terraform/tests/resource_gkehub_feature_test.go.erb

```diff
@@ -0,0 +1,1092 @@
+      },
+      {
+        ResourceName:      "google_gkehub_feature_membership.feature_member_2",
+        ImportState:       true,
+        ImportStateVerify: true,
+      },
+    },
+  })
+}
+
+func testAccGKEHubFeature_gkehubFeatureAcmUpdateStart(context map[string]interface{}) string {
+  return projectSetup(context) + Nprintf(`
+resource "google_container_cluster" "primary" {
+  name               = "tf-test-cl%{random_suffix}"
+  location           = "us-central1-a"
+  initial_node_count = 1
+  project = google_project.project.project_id
+  provider = google-beta
+  depends_on = [google_project_service.mci, google_project_service.container, google_project_service.container, google_project_service.gkehub]
+}
+
+resource "google_container_cluster" "secondary" {
+  name               = "tf-test-cl2%{random_suffix}"
+  location           = "us-central1-a"
+  initial_node_count = 1
+  project = google_project.project.project_id
+  provider = google-beta
+  depends_on = [google_project_service.mci, google_project_service.container, google_project_service.container, google_project_service.gkehub]
+}
+
+resource "google_container_cluster" "tertiary" {
+  name               = "tf-test-cl3%{random_suffix}"
+  location           = "us-central1-a"
+  initial_node_count = 1
+  project = google_project.project.project_id
+  provider = google-beta
+  depends_on = [google_project_service.mci, google_project_service.container, google_project_service.container, google_project_service.gkehub]
+}
+
+resource "google_gke_hub_membership" "membership" {
+  project = google_project.project.project_id
+  membership_id = "tf-test1%{random_suffix}"
+  endpoint {
+    gke_cluster {
+      resource_link = "//container.googleapis.com/${google_container_cluster.primary.id}"
+    }
+  }
+  description = "test resource."
+  provider = google-beta
+}
+
+resource "google_gke_hub_membership" "membership_second" {
+  project = google_project.project.project_id
+  membership_id = "tf-test2%{random_suffix}"
+  endpoint {
+    gke_cluster {
+      resource_link = "//container.googleapis.com/${google_container_cluster.secondary.id}"
+    }
+  }
+  description = "test resource."
+  provider = google-beta
+}
+
+resource "google_gke_hub_membership" "membership_third" {
+  project = google_project.project.project_id
+  membership_id = "tf-test3%{random_suffix}"
+  endpoint {
+    gke_cluster {
+      resource_link = "//container.googleapis.com/${google_container_cluster.tertiary.id}"
+    }
+  }
+  description = "test resource."
+  provider = google-beta
+}
+
+resource "google_gkehub_feature" "feature" {
+  project = google_project.project.project_id
+  name = "configmanagement"
+  location = "global"
+
+  labels = {
+    foo = "bar"
+  }
+  provider = google-beta
+  depends_on = [google_project_service.mci, google_project_service.container, google_project_service.container, google_project_service.gkehub]
+}
+
+resource "google_gkehub_feature_membership" "feature_member_1" {
```

Comment:
Can we do `google_gkehub_feature_membership` tests in a separate file from `google_gkehub_feature` tests? They're very connected so I see why it makes sense to do both in the same one, but that's consistent with other parent/child resources like GKE cluster + pool.